addons/l10n_ar/models/account_tax_group.py

```diff
@@ -30,3 +31,23 @@ class AccountTaxGroup(models.Model):
         ('8', '5%'),
         ('9', '2,5%'),
     ], string='VAT AFIP Code', index=True, readonly=True)
+
+    @api.ondelete(at_uninstall=False)
+    def check_uninstall_required(self):
+        """
+        Make sure we don't uninstall a required tax group
+        """
+        ar_companies = self.filtered(lambda g: g.company_id.chart_template.startswith('ar_')).mapped('company_id')
```

Comment:
why not filtering on country code == 'AR', as we commonly do? :thinking:

Reply:
If an AR company does not use our CoA, it's very unlikely it will have this specific XML-ID (`account.{company.id}_tax_group_percepcion_ganancias`) for this specific tax group. If is uses our CoA, we can be pretty sure it should have this tax group. This tax group is used in our Argentinean localization to generate the electronic invoice e.g.

Reply:
ok you have a point :)